### PR TITLE
EPAS 17: version bumps

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?

https://enterprisedb.atlassian.net/browse/DOCS-1097 

Bumping v16 references to v17. 
Left out mentions in release notes sections, as those will be removed for the 17.0.0 release.

PS: The link checker is warning about a link that is broken in the branch, but that I believe will resolve successfully when we release, hence why I didn't correct it.

### CODE REVIEW

File: product_docs/docs/epas/17/epas_security_guide/02_protecting_against_sql_injection_attacks/02_configuring_sql_protect.mdx [hunk 1]
```diff
@@ -127,9 +127,9 @@ For each database that you want to protect, you must determine the roles you wan
 <!-- Consider replacing -->
```
Comment (nidhibhammar) [inline product_docs/docs/epas/17/epas_security_guide/02_protecting_against_sql_injection_attacks/02_configuring_sql_protect.mdx:127]: @tusharahuja, can you help with the new example here?

File: product_docs/docs/epas/17/epas_security_guide/02_protecting_against_sql_injection_attacks/04_backing_up_restoring_sql_protect.mdx [hunk 2]
```diff
@@ -81,9 +81,9 @@ CREATE SCHEMA
   This deletion removes any existing rows in the `edb_sql_protect_rel` table that were backed up from the original database. These rows don't contain the correct OIDs relative to the database where the backup file was restored.
 <!--Consider replacing -->
```
Comment (nidhibhammar) [inline product_docs/docs/epas/17/epas_security_guide/02_protecting_against_sql_injection_attacks/04_backing_up_restoring_sql_protect.mdx:82]: @tusharahuja, can you help with the new example here?

**Comment by nidhibhammar** [comment]
Note to self - I have asked for the new examples from Tushar, which need to be replaced. Rest all the changes looks good to me.

**Comment by nidhibhammar** [comment]
@gvasquezvargas, the version number to be released is 17.1.0. We need to update it.

File: product_docs/docs/epas/17/epas_security_guide/02_protecting_against_sql_injection_attacks/02_configuring_sql_protect.mdx (hunk 1 above)
Comment (tusharahuja) [reply product_docs/docs/epas/17/epas_security_guide/02_protecting_against_sql_injection_attacks/02_configuring_sql_protect.mdx:127]: The example is already there in the doc 

edb=# SELECT protect_role('appuser');
Output
 protect_role
--------------

(1 row)

**Review by tusharahuja — COMMENTED** [review]
Hi,
examples already there in the doc. please let me know if any more information is needed. Thanks.

**Comment by nidhibhammar** [comment]
@gvasquezvargas, we can bump the version in both examples and remove the comment enclosed in angular brackets.

**Pull request merged**